recipes/qt/5.x.x/conanfile.py

```diff
@@ -507,9 +507,7 @@ def build(self):
             args += ["-opengl dynamic"]
 
         if self.options.get_safe("with_vulkan", False):
-            args.append("-vulkan")
-        else:
```

Comment:
We have to keep the else to disable explicitly vulkan, even in cases where qt detects system vulkan